Contentful.AspNetCore/TagHelpers/ContentfulAssetTagHelper.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using Contentful.Core;
+using Contentful.Core.Images;
+using System.Threading.Tasks;
+
+namespace Contentful.AspNetCore.TagHelpers
+{
+    public class ContentfulImageTagHelper : TagHelper
+    {
+        private readonly IContentfulClient _client;
+
+        public ContentfulImageTagHelper(IContentfulClient client)
+        {
+            _client = client;
+        }
+
+        public string AssetId { get; set; }
+        public string Url { get; set; }
+        public int Width { get; set; }
+        public int Height { get; set; }
+        public ImageFormat Format { get; set; }
+        public ImageResizeBehaviour ResizeBehaviour { get; set; }
+        public ImageFocusArea FocusArea { get; set; }
+        public int? JpgQuality { get; set; }
+        public bool ProgressiveJpg { get; set; }
+        public int? CornerRadius { get; set; }
+        public string BackgroundColor { get; set; }
```

Comment:
Initally no, but what I would like to do is to add a regular expression validation to the field. Making sure it's either #cc0000 or rgb:cc0000 or similar. This would also give contextual help when a user is trying to type it in Visual Studio.